db/seeds.rb

```diff
@@ -1,7 +1,42 @@
+require 'csv'    
+require 'byebug'
+
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# HazardCategories
+csv_text = File.read(Rails.root.join('db', 'seed_data', 'HazardCategories.csv'))
+csv = CSV.parse(csv_text, :headers => true)
+csv.each do |row|
+  HazardCategory.find_or_create_by!(row.to_hash)
+end
+
+#Control methods
+HazardControlMethod.find_or_create_by!(name: 'Eliminate')
+HazardControlMethod.find_or_create_by!(name: 'Isolate')
+HazardControlMethod.find_or_create_by!(name: 'Minimise')
+
+#Hazards
+csv_text = File.read(Rails.root.join('db', 'seed_data', 'Hazards.csv'))
+csv = CSV.parse(csv_text, :headers => true)
+csv.each do |row|
+    next if Hazard.exists?(reference_number: row['reference_number'])
```

Comment:
💡 I prefer not to do breaks in loops myself. Something to potentially use next time is the `.select` or `.reject` enumerable method next time prior to this loop, if performance is a concern `.lazy` can be chained in there as well.